templates/crate/details.html

```diff
@@ -75,15 +75,12 @@
                                     {# If the repo link is for github, show some github stats #}
                                     {# TODO: add support for hosts besides github (#35) #}
                                     {%- if details.github_metadata -%}
-                                        {{ "github" | fab(fw=true) }}
                                         {{ "star" | fas(fw=true) }} {{ details.github_metadata.stars }}
                                         {{ "code-branch" | fas(fw=true) }} {{ details.github_metadata.forks }}
-                                        {{ "exclamation-circle" | fas(fw=true) }} {{ details.github_metadata.issues }}
-
-                                    {# If the repo link is unknown, just show a normal link #}
-                                    {%- else -%}
-                                        {{ "code-branch" | fas(fw=true) }} Repository
+                                        {{ "exclamation-circle" | fas(fw=true) }} {{ details.github_metadata.issues }}<br>
                                     {%- endif -%}
+
+                                    {{ "code-branch" | fas(fw=true) }} Repository
                                 </a>
```

Comment:
This code is duplicated two times: one in the dropdown of rustdoc pages and one here. It would probably be good to extract it into a macro, so changes can be shared between the two places.

Reply:
Oh wow, didn't even pay attention to that. I'll do that too then.

Reply:
Ok, tested and the result is pretty bad. It gets even worse on long names. So in the end, I did the opposite: I simplified it like when you don't have github information.

Reply:
Missed this comment... how did it look?

Reply:
The github logo is on its own line and the name is on a second one. And the repository info are just too much. (Don't have the screenshot anymore, urg)

Reply:
We can use some rules to have ellipsis on the repository name if it's too long:

```css
overflow: hidden;
text-overflow: ellipsis;
white-space: nowrap;
```

![2021-01-07--11-54-25](https://user-images.githubusercontent.com/2299951/103884597-1c708f80-50df-11eb-852b-92f91b12426b.png)

Reply:
True, but in my opinion, this still remains a bit too much.

Reply:
Personally I would be ok with just showing 'Repository'. I've heard lots of feedback saying that it's not very obvious right now.

If you want to show the stars/PRs/issues underneath that seems fine, but showing the repository name too seems like overkill.